src/libponyrt/ds/hash.h

```diff
@@ -26,7 +26,7 @@ PONY_EXTERN_C_BEGIN
 
 /** Definition of a hash map entry for uintptr_t.
  */
-typedef struct hashmap_entry_t
```

Comment:
I think this was a stray erroneous change from a misrouted keyboard press into my editor. It should probably be removed.